src/components/Common/PlayButton/index.tsx

```diff
@@ -43,18 +46,14 @@ const PlayButton: React.FC<PlayButtonProps> = ({ links }) => {
           <span>{links[0].text}</span>
         </>
       }
-      onClick={() => {
-        window.open(links[0].url, '_blank');
-      }}
     >
       {links.length > 1 &&
         links.slice(1).map((link, i) => {
           return (
             <ButtonWithDropdown.Item
               key={`play-button-dropdown-item-${i}`}
-              onClick={() => {
-                window.open(link.url, '_blank');
-              }}
+              href={link.url}
+              target="_blank"
               buttonType="ghost"
```

Comment:
I feel like this is the component that should support a-tags?

Reply:
It actually already is an `a` tag.